src/R/Editor/Impl/Data/WorkspaceVariableProvider.cs

```diff
@@ -85,7 +85,7 @@ internal sealed class WorkspaceVariableProvider : RSessionChangeWatcher, IVariab
                 }
 
                 // May be a package object line mtcars$
-                variableName = TrimToTrailingSelector(variableName);
+                variableName = TrimToFirstSelector(variableName);
```

Comment:
The problem is that `exists()` was added to avoid that error message in REPL when variable doesn't exist. Can't find the original bug... But let me try some varieties.

And yes, we have task for the cleanup